client/src/components/advancedSelectWidget/AdvancedSelect.js

```diff
@@ -372,7 +374,7 @@ const AdvancedSelect = ({
                 <InputGroup>
                   <Form.Control
                     name={fieldName}
-                    value={searchTerms || ""}
+                    value={searchTerms}
```

Comment:
Revert this change; if `searchTerms` is `undefined` or `null`, the form control will have no value (which is an error, and `undefined` will even make it an uncontrolled input which you can't give a value).